app/models/amusement_park.rb

```diff
@@ -1,3 +1,7 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
-end
\ No newline at end of file
+
+  def unique_mechanics
+    rides.map{|ride| ride.mechanics}.flatten.uniq #couldn't find a better way to do this with AR in time
```

Comment:
The easy way: Add an additional association!

Since an amusement park has many rides, you can leverage this association and add:
`has_many :mechanics, through: :rides`
This will allow you to do: `self.mechanics.distinct`

The harder way: Do a join
`rides.joins(:mechanics).select("DISTINCT mechanics.name")  ## you could also do .distinct instead of including it in the select`